pkgs/development/python-modules/future/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib
+, pkgs
+, buildPythonPackage
+, fetchPypi
+, isPy26
+}:
+
+buildPythonPackage rec {
+  pname = "future";
+  version = "0.16.0";
+  name = "${pname}-${version}";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1nzy1k4m9966sikp0qka7lirh8sqrsyainyf8rk97db7nwdfv773";
+  };
+
+  propagatedBuildInputs = lib.optionals isPy26 (with pkgs.pythonPackages; [ importlib argparse ]);
```

Comment:
We are not going to use the package set, but we're going to pass in the individual requirements. Therefore, in the top, you can add the parameters importlib and argparse and remove here `with pkgs.pythonPackages; `